src/Factory/NonceFactoryinterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace RebelCode\WordPress\Nonce\Factory;
+
+use RebelCode\WordPress\Nonce\NonceInterface;
+
+/**
+ * Something that can create nonce instances.
+ *
+ * @since [*next-version*]
+ */
+interface NonceFactoryinterface
+{
+    /**
+     * Creates a new nonce instance.
+     *
+     * @since [*next-version*]
+     *
+     * @param string $id    Option nonce ID.
+     * @param array $config Optional additional configuration data.
+     *
+     * @return NonceInterface
+     */
+    public function make($id = '', $config = []);
```

Comment:
I think the ID should go in the `$config`, which should be the only parameter, and an optional one, as always.